projects/online/online/utils/gdb.py

```diff
@@ -77,12 +77,21 @@ def submit_low_latency_pe(
         skymap.writeto(skymap_fname)
 
         logging.debug("Submitting skymap to GraceDB")
-        self.write_log(graceid, "skymap", filename=skymap_fname, tag_name="pe")
+        self.write_log(
+            graceid,
+            "skymap",
+            filename=skymap_fname,
+            tag_name=["sky_loc", "SKYMAP_READY"],
```

Comment:
I think `SKYMAP_READY` needs to be a `label` . See [here](https://ligo-gracedb.readthedocs.io/en/latest/api.html#ligo.gracedb.rest.GraceDb.write_log): the `write_log` has a `label` argument as well

Reply:
Ah, missed that. I'll fix those.